task.sql

```diff
@@ -1,3 +1,17 @@
 USE ShopDB; 
 
 -- Create your stored procedure here
+
+DELIMITER //
+CREATE PROCEDURE get_warehouse_product_inventory(
+  IN WarehouseID INT
+)
+
+BEGIN
+  SELECT Products.Name, ProductInventory.WarehouseAmount
+  FROM Products
+  INNER JOIN ProductInventory
+  ON Products.ID = ProductInventory.ProductID
+  WHERE ProductInventory.WarehouseID = WarehouseID;
+END //
+DELIMITER ;
```

Comment:
![image](https://github.com/user-attachments/assets/23a3d71b-1ca3-4c36-9075-b68d0c8e6f31)

Add Eof(End-of-file), need to add a blank line at the end